src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --header-bg: #fff;
+  --header-height: 60px;
+  --nav-link-color: #000;
+  --nav-link-hover-color: #00acdc;
+  --nav-font-size: 12px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--header-bg);
+  width: 100%;
+  padding: 0 50px;
+}
+
+.logo_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  gap: 20px;
+}
+
+.nav_link {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--nav-link-color);
+  font-family: Roboto, sans-serif;
+  font-size: var(--nav-font-size);
+  font-weight: 500;
+  height: var(--header-height);
+  line-height: 14.6px;
+}
+
+.nav_link:hover {
+  color: var(--nav-link-hover-color);
+}
+
+.is-active {
+  color: var(--nav-link-hover-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background: var(--nav-link-hover-color);
+  position: absolute;
```

Comment:
The `position: absolute;` property for the `.is-active::after` pseudo-element requires that the parent element has a `position` property set to `relative`, `absolute`, or `fixed` to ensure proper positioning. Ensure that the parent element of `.is-active` has a suitable `position` property.